src/api/module/history/entity/index.ts

```diff
@@ -16,7 +18,7 @@ export type History = {
 export type Payload = {
   historyType: number | undefined;
   juz?: number | undefined;
-  surah?: number | undefined;
+  surah?: Option[] | number | undefined;
```

Comment:
is it good to have different type of types? array and number? not confusing?

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/20/commits/14fffc6b84cef20f29f3b26d16f1822aa99d6d20, thank you